src/util/ZoneUtil.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef UTIL_ZONEUTIL_H_
+#define UTIL_ZONEUTIL_H_
+
+#include "common/base/Base.h"
+#include "common/base/StatusOr.h"
+#include "parser/MaintainSentences.h"
+
+namespace nebula {
+namespace graph {
+
+class ZoneUtil final {
```

Comment:
What's the plan to do with the `ZoneUtil` ans `GroupUtil`?

Reply:
I  will move some zone and group assign code into it.